core/src/main/java/org/commonjava/indy/core/content/DefaultContentManager.java

```diff
@@ -311,7 +311,16 @@ private boolean checkListingMask( final ArtifactStore store, final String path )
 
         AbstractRepository repo = (AbstractRepository) store;
         Set<String> maskPatterns = repo.getPathMaskPatterns();
-        logger.debug( "Checking mask in: {}, type: {}, patterns: {}", repo.getName(), repo.getKey().getType(), maskPatterns );
+
+        if ( logger.isTraceEnabled() )
+        {
+            logger.trace( "Checking mask in: {}, type: {}, patterns: {}", repo.getName(), repo.getKey().getType(),
+                          maskPatterns );
+        }
+        else
+        {
+            logger.debug( "Checking mask in: {}, type: {}", repo.getName(), repo.getKey().getType() );
+        }
```

Comment:
I think this is a result of my merging @yma88 PR #725  for enabling readonly locations. It might be worthwhile to help her diagnose / fix the failing tests in that. I should have waited on the Galley PR merge that corresponded to it.